mne/io/eeglab/eeglab.py

```diff
@@ -596,6 +630,28 @@ def __init__(self, input_fname, events=None, event_id=None, tmin=0,
         logger.info('Ready.')
 
 
+def _check_boundary(annot, event_id):
+    if event_id is None:
+        event_id = dict()
+    if "boundary" in annot.description and "boundary" not in event_id:
```

Comment:
another small nitpick: the boundary events at the edges of the signal (before the first sample or after the last one) are obviously benign as the only signal (in eeglab at least) that the signal was cut out of a longer continuous file. Ideally we would not warn in such cases. But that may be in some other PR.

Reply:
subsequent PR (@mmagnuski can you open an issue?)